src/elm/Main.elm

```diff
@@ -1187,14 +1187,14 @@ viewContent model =
             ( String.join "/" [ org, repo ] ++ " builds" ++ page
             , div []
                 [ Pager.view model.builds.pager Pager.defaultLabels GotoPage
-                , Pages.Builds.view model.builds model.time org repo
+                , lazy4 Pages.Builds.view model.builds model.time org repo
```

Comment:
although `time` in the main model is updated every few seconds via the `subscriptions` I still added `lazy` here for consistency and potential performance updates. (AKA, it couldnt hurt, right?)